picard/tagger.py

```diff
@@ -159,7 +160,7 @@ class Tagger(QtWidgets.QApplication):
     _debug = False
     _no_restore = False
 
-    def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
+    def __init__(self, picard_args, unparsed_args, localedir, autoupdate, dark_theme=False):
```

Comment:
Just a minor thing, instead of passing a boolean, it would be more future-proof to pass a numeric constant.
Something like:

```python
from enum import IntEnum, unique

@unique
class Themes(IntEnum):
    default = 0
    dark = 1
    sometheme = 2

def __init__(self, picard_args, unparsed_args, localedir, autoupdate, theme=Themes.default):

...
if theme == Themes.dark:
    ...

```

It will also allow us to implement a theme chooser, and to save/load user prefs.

Reply:
I'm going to close this PR. Philipp's branch is way ahead on that. Trying palettes for the script highlighting.



<details>
   <summary>
   My favorite one so far
   </summary>

![image](https://user-images.githubusercontent.com/1678454/81761211-bd02a700-949f-11ea-9f52-71a4ba32e4ec.png)

![image](https://user-images.githubusercontent.com/1678454/81761284-f3402680-949f-11ea-8d13-1930587d42b0.png)

</details>